examples/notebook/tensorflow/vgg19_ibean/inc_quantize_vgg19.ipynb

```diff
@@ -364,22 +346,28 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from neural_compressor.experimental import Quantization, common\n",
-    "\n",
-    "def preprocess_1(image, label):\n",
+    "def preprocess(image, label):\n",
     "    image = tf.cast(image, tf.float32)/255.0\n",
-    "    return tf.image.resize(image, [w, h]), label\n",
+    "    return  tf.image.resize(image, [w, h]), np.array(tf.one_hot(label, class_num))    \n",
+    "\n",
     "\n",
+    "def load_raw_dataset():\n",
+    "    raw_datasets, raw_info = tfds.load(name = 'beans', with_info = True,\n",
+    "                                       as_supervised = True, \n",
+    "                                       split = ['train', 'test'])\n",
+    "                                       \n",
+    "    return raw_datasets, raw_info\n",
+    "    \n",
     "class Dataset(object):\n",
     "    def __init__(self):\n",
     "        datasets , info = load_raw_dataset()        \n",
-    "        self.train_dataset = [preprocess_1(v, l) for v,l in datasets[0]]\n",
+    "        self.test_dataset = [preprocess(v, l) for v,l in datasets[-1]]\n",
```

Comment:
original code use train_dataset. It will help get good accuracy.
please use train_dataset.

Reply:
Updated again now.